terraform-modules/cur-setup-source/main.tf

```diff
@@ -68,6 +68,10 @@ resource "aws_s3_bucket_lifecycle_configuration" "this" {
   }
 }
 
+data "aws_partition" "current" {}
+data "aws_region" "current" {}
+data "aws_caller_identity" "current" {}
+
```

Comment:
All except the aws_partition data source duplicates of data sources at the top of the file. Recommend moving the aws_partition data source to the top and removing the others to avoid duplicates (Terraform will fail).